tests/local/network.go

```diff
@@ -174,11 +163,61 @@ func newLocalNetwork(warpGenesisFile string) *localNetwork {
 	}
 	res.setSubnetValues(subnetAID)
 	res.setSubnetValues(subnetBID)
-	res.setSubnetValues(subnetCID)
+	res.setPrimaryNetworkValues()
 	return res
 }
 
-func (n *localNetwork) setSubnetValues(subnetID ids.ID) {
+// Should be called after setSubnetValues for all subnets
+func (n *LocalNetwork) setPrimaryNetworkValues() {
+	var nodeURIs []string
+	nodeURIs = append(nodeURIs, n.subnetsInfo[n.subnetAID].NodeURIs...)
+	nodeURIs = append(nodeURIs, n.subnetsInfo[n.subnetBID].NodeURIs...)
```

Comment:
Do you know if there are any other nodes in the local primary network other than the subnet validator nodes? 

If not, could you add a comment explaining that all nodes should be accounted for here?

Reply:
Nope, I believe these should be all the nodes. I added a comment noting such.

For context, we only ever use the 1st element of this list, so the actual length and content is pretty arbitrary.